### PR TITLE
add font color option

### LINKED ISSUE #18: Support customizing the font color

### PR DESCRIPTION
**Pull request by eric-kimbrel**
fixes #18

### CODE REVIEW

File: packages/attribute-slicer/src/AttributeSlicer.ts [hunk 1]
```diff
@@ -165,6 +165,20 @@ export class AttributeSlicer {
         }
     }
 
+    private _itemTextColor = "#000";
```
Comment (stopyoukid) [inline packages/attribute-slicer/src/AttributeSlicer.ts]: Comments por favor.

File: packages/attribute-slicer-powerbi/src/state.ts [hunk 2]
```diff
@@ -146,6 +147,13 @@ export default class AttributeSlicerVisualState extends HasSettings implements I
     })
     public textSize?: number;
 
+   @color({
```
Comment (stopyoukid) [inline packages/attribute-slicer-powerbi/src/state.ts]: Comments por favor.

File: packages/attribute-slicer/src/SlicerItem.tmpl.ts [hunk 3]
```diff
@@ -30,11 +30,13 @@ import * as d3 from "d3";
...
  */
-export default function (item: SlicerItem, sizes: { category: number; value: number }, alignTextLeft: boolean, showValueLabels?: boolean) {
+export default function (item: SlicerItem, sizes: { category: number; value: number }, alignTextLeft: boolean,
+        showValueLabels?: boolean, itemTextColor?: string) {
     "use strict";
     const { match, matchPrefix, matchSuffix, valueSegments, renderedValue } = item;
     const alignStyle = alignTextLeft ? "text-align:left;" : "";
-    const categoryStyle = `display:inline-block;overflow:hidden;max-width:${sizes.category}%;${alignStyle}`;
+    const itemFontColor = itemTextColor ? itemTextColor : "#000";
+    const categoryStyle = `display:inline-block;overflow:hidden;max-width:${sizes.category}%;${alignStyle};color:${itemFontColor}`;
```
Comment (stopyoukid) [inline packages/attribute-slicer/src/SlicerItem.tmpl.ts]: I forgot, could you add tests to SlicerItem.spec.ts for the three params that was added?

Comment (eric-kimbrel) [inline packages/attribute-slicer/src/SlicerItem.tmpl.ts]: yeah, i'll get those added.

**Pull request merged**